src/Display/Swipeable/Swipeable.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+
+import { Swipeable, Props } from './Swipeable';
+import { SwipeableCardExample } from './SwipeableStyle';
+
+export default {
+  title: 'Display/Swipeable',
+  component: Swipeable,
+  argTypes: {
+    className: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} as Meta;
+
+const Template: Story<Props> = ({ ...args }) => {
+  return (
+    <>
+      <Swipeable {...args}>
+        {Array.from(Array(10).keys()).map(number => (
+          <Swipeable.Item key={number}>
+            <SwipeableCardExample>
+              <h1>Card. {number}</h1>
+            </SwipeableCardExample>
+          </Swipeable.Item>
+        ))}
+      </Swipeable>
+    </>
```

Comment:
What's the Fragment for?

Reply:
It's 2 have 10 cards as Swipeable Item. 
Didn't understand the comment I think 🤔

Reply:
I mean the `<>...</>`. Usually that's only used because you need to return a single element in React. But `<Swipeable>` could be that element here.

Reply:
Ooh I didn't notice them after the comment too, I was like there's no Fragment so why the comment :P . Yes, you're right. It's not required.